docs/src/overview/isolated-kubernetes.md

```diff
@@ -0,0 +1,401 @@
+
+For application workloads please see below how these images must be provided.
+
+## Flavors
+
+With the introduction of Isolated Kubernetes Clusters, the cluster user must decide upon cluster creation which type of Isolation he needs for his workload.
+There are 3 different flavours available:
+
+- Internet Access Baseline: This is the default cluster creation mode, which does not change any aspects of network and registry access.
+- Internet Access Forbidden: No internet access is possible, neither ingress not egress.
+- Internet Access Restricted: No internet access is possible, neither ingress not egress, but can be enabled by the cluster user.
+
+Please see the detailed description of these flavours below.
+
+### Internet Access Baseline
+
+This is the default configuration of a kubernetes cluster, egress traffic is controlled by multiple CWNP's (Cluster Wide Network Policy), ingress traffic is possible by deploying a Service Type Loadbalancer. The cluster user can add additional CWNP's without any restrictions and is responsible for them.
+
+Container images can be pulled from any reachable container registry.
+
+DNS and NTP are configured to internet DNS Resolvers and NTP Servers
+
+The list of CWNP's which are deployed by default are:
+
+```bash
+> kubectl get clusterwidenetworkpolicies.metal-stack.io
+NAME                 STATUS     MESSAGE
+allow-to-http        deployed
+allow-to-https       deployed
+allow-to-apiserver   deployed
+allow-to-dns         deployed
+allow-to-ntp         deployed
+allow-to-registry    deployed
+allow-to-storage     deployed
+allow-to-vpn         deployed
+```
+
+The purposes of this CWNPs are:
+
+| Rule Name          | Isolation            | Destination                                          | Purpose                                                                                      |
+|--------------------|----------------------|------------------------------------------------------|----------------------------------------------------------------------------------------------|
+| allow-to-http      | baseline             | 0.0.0.0/0                                            | egress via http                                                                              |
+| allow-to-https     | baseline             | 0.0.0.0/0                                            | egress via https                                                                             |
+| allow-to-apiserver | restricted, forbidden | IP of the Kubernetes API Server on the control plane | API Server communication of kubelet and other controllers                                    |
+| allow-to-dns       | all                  | IP of the private DNS Server                         | DNS resolution from the Kubernetes worker nodes and containers                               |
```

Comment:
This sounds like DNS and NTP, even for baseline clusters, is restricted to the private DNS and NTP in the infrastructure cluster. Is that actually the case?

Reply:
I have to change this table because confusing